metricbeat/module/prometheus/remote_write/_meta/docs.asciidoc

```diff
@@ -22,7 +22,17 @@ However this will increase the memory usage of Metricbeat.
 - `capacity`: It is recommended to set capacity to 3-5 times `max_samples_per_send`.
 Capacity sets the number of samples that are queued in memory per shard, and hence capacity should be high enough so as to
 be able to cover `max_samples_per_send`.
-
+- `write_relabel_configs`: It is a relabeling, that applies to samples before sending them to the remote endpoint. This could be used to limit which samples are sent.
```

Comment:
Interesting! So this could be used to avoid overloading Metricbeat when Prometheus sends lots of metrics, right?

Reply:
yes, I took this from official documentation https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write (first sentence), I thought it could be useful to add it in tips here

Reply:
cc @ruflin @ishleenk17 FYI

Reply:
This definitively sounds interesting. It would be nice to provide an example on how this config could look so anyone that wants to use it, would not have to search for some samples (which I just did). Worth also adding a link to the official docs here.

Reply:
below - there is an example for this configuration - https://github.com/elastic/beats/pull/34163/files#diff-8ab9ea7fd7d76fa5a72d7eb71845defcb6c5b70f6e49dd9e9356b1fbaa40645eR27-R35
or do you want to see something more specific?